python/pyspark/pandas/indexes/base.py

```diff
@@ -289,7 +289,7 @@ def _summary(self, name: Optional[str] = None) -> str:
 
         if name is None:
             name = type(self).__name__
-        return "%s: %s entries%s" % (name, total_count, index_summary)
+        return "%s: %s entries%s" % (name, int(total_count), index_summary)
```

Comment:
qq: why do we need to fix here?

Reply:
When I was writing tests for the info method I found out that it prints different outputs and the difference was that pandas return:
```
Index: 9 entries, 0.6658222030375259 to 0.7862583284518239
Data columns (total 2 columns):
 #   Column  Non-Null Count  Dtype
---  ------  --------------  -----
 0   a       9 non-null      int64
 1   b       9 non-null      int64
```
but spark pandas api return 
```
Index: 9.0 entries, 0.6658222030375259 to 0.7862583284518239
Data columns (total 2 columns):
 #   Column  Non-Null Count  Dtype
---  ------  --------------  -----
 0   a       9 non-null      int64
 1   b       9 non-null      int64
```
So to align outputs I've done the fix

Reply:
Got it!